src/Connector.js

```diff
@@ -162,6 +167,11 @@ class Connector {
     this.onDataUpdatedCb = cb;
   }
 
+  // cb(event) where event is { id }
+  async onDeviceUnregistered(cb) {
+    this.onUnregisteredCb = cb;
```

Comment:
Please, change to `onDeviceUnregisteredCb` to keep following the pattern of the other callbacks.